src/Definition/Redshift.php

```diff
@@ -58,9 +58,14 @@ public function getCompression()
     public function getSQLDefinition()
     {
         $definition =  $this->getType();
-        if ($this->getLength() && $this->getLength() != "") {
-            $definition .= "(" . $this->getLength() . ")";
+
+        $typesWithoutLength = ['TIMESTAMP', 'DATE', 'FLOAT', 'BOOLEAN'];
+        if (!in_array($this->getBasetype(), $typesWithoutLength)) {
```

Comment:
Akorat, kdyz to pak pouzijes v query na vytvoreni tabulky, tak to skonci chybou.